app/controllers/concerns/organization_scopable.rb

```diff
@@ -44,4 +51,12 @@ def after_sign_in_path_for(resource_or_scope)
   def after_sign_out_path_for(resource_or_scope)
     root_path
   end
+
+  def new_person_after_sign_up_path
+    if Current.organization.external_form_url
```

Comment:
let's add `.present` here just to be safe. I believe we have a validation to prevent users from saving an empty string, but it might still be possible. So, this will make sure there's actually a string/url